releasenotes/notes/features-5291b4198f98952c.yaml

```diff
@@ -0,0 +1,6 @@
+---
+
+features:
+  - |
+    Updating process agent discovery configuration to include a Data Scrubber for obfuscating sensitive information such as "passwords", keys, etc.
```

Comment:
I don't think it should be quoted here, can be drop "etc." and just say, passwords, API keys, or tokens. I believe we used to capitalize Process Agent in release notes whereas data scrubber doesn't need to be capitalized. cc @jhgilbert since you reviewed previously.

Reply:
@xornivore I have updated the message following your suggestions. However I left the "process agent" in lower case following previous process discovery's PRs [#9082](https://github.com/DataDog/datadog-agent/pull/9082) and [#20880](https://github.com/DataDog/datadog-agent/pull/20880)